### PR TITLE
OWLS-90664,Added test to check the order of loading images in AI

### PR DESCRIPTION
**Pull request by marinakog**
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/5622/

### CODE REVIEW

**Review by xiancao — APPROVED** [review]

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java [hunk 1]
```diff
@@ -195,6 +201,10 @@ public void testCreateDomainUsingMultipleAuxiliaryImages() {
     // create models dir and copy model, archive files if any
     Path modelsPath2 = Paths.get(multipleAIPath2.toString(), "models");
     assertDoesNotThrow(() -> Files.createDirectories(modelsPath2));
+    Path multipleAIPathToFile2 = Paths.get(RESULTS_ROOT, "multipleauxiliaryimage2/test.txt");
+    String content2 = "2";
+    assertDoesNotThrow(() -> Files.write(multipleAIPathToFile2, content2.getBytes()),
```
Comment (sankarpn) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java:206]: Can you delete the file if it exists?

**Review by sankarpn — CHANGES_REQUESTED** [review]
Just one comment

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java (hunk 1 above)
Comment (marinakog) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java:206]: we always delete whole directory before creating anything, see below code:
assertDoesNotThrow(() -> FileUtils.deleteDirectory(multipleAIPath2.toFile()));
    assertDoesNotThrow(() -> Files.createDirectories(multipleAIPath2));

    // create models dir and copy model, archive files if any
    Path modelsPath2 = Paths.get(multipleAIPath2.toString(), "models");
    assertDoesNotThrow(() -> Files.createDirectories(modelsPath2));
    Path multipleAIPathToFile2 = Paths.get(RESULTS_ROOT, "multipleauxiliaryimage2/test.txt");

Comment (sankarpn) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java:206]: lg

**Review by sankarpn — APPROVED** [review]
lgtm

**Pull request merged**